inst/qview/qview.R

```diff
@@ -0,0 +1,92 @@
+library(shiny)
+library(tidyverse)
+library(DT)
+
+
+#' queueViewOutput
+#' @export
```

Comment:
just FYI in `inst/` nothing gets doc'd by roxygen. I absolutely agree with using roxygen comments in general for the code, but just know that this stuff isn't actually exported until its in `R/`

Reply:
I was actually confused why we're putting this in inst -- from what I can gather, using @export is how you'd actually make the module accessible. We can talk about this next time we catch up.

Reply:
this doesn't need to be in inst, the module functions can/should be regular functions in `R/` to be accessible from the package directly